modules/Microsoft.HealthcareApis/workspaces/.test/common/dependencies.bicep

```
@@ -0,0 +1,77 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+@description('Required. The name of the Event Hub Namespace to create.')
+param eventHubNamespaceName string
+
+@description('Required. The name of the Event Hub consumer group to create.')
+param eventHubConsumerGroupName string
+
+@description('Required. The name of the Storage Account to create.')
+param storageAccountName string
+
+resource managedIdentity 'Microsoft.ManagedIdentity/userAssignedIdentities@2018-11-30' = {
+    name: managedIdentityName
+    location: location
+}
+
+resource storageAccount 'Microsoft.Storage/storageAccounts@2022-05-01' = {
+    name: storageAccountName
+    location: location
+    sku: {
+        name: 'Standard_LRS'
+    }
+    kind: 'StorageV2'
+}
+
+resource ehns 'Microsoft.EventHub/namespaces@2022-01-01-preview' = {
+    name: eventHubNamespaceName
+    location: location
+    sku: {
+        name: 'Standard'
+        tier: 'Standard'
+        capacity: 1
+    }
+    properties: {
+        zoneRedundant: false
+        isAutoInflateEnabled: false
+    }
+}
+
+resource ehub 'Microsoft.EventHub/namespaces/eventhubs@2022-01-01-preview' = {
+    name: '${eventHubNamespaceName}-hub'
+    parent: ehns
+    properties: {
+        messageRetentionInDays: 1
+        partitionCount: 1
+    }
+}
+
+resource ehub_consumergroup 'Microsoft.EventHub/namespaces/eventhubs/consumergroups@2022-01-01-preview' = {
+    name: eventHubConsumerGroupName
+    parent: ehub
+}
```

Just a suggestion. Feel free to discregard if you don't like it.

```suggestion
resource ehns 'Microsoft.EventHub/namespaces@2022-01-01-preview' = {
    name: eventHubNamespaceName
    location: location
    sku: {
        name: 'Standard'
        tier: 'Standard'
        capacity: 1
    }
    properties: {
        zoneRedundant: false
        isAutoInflateEnabled: false
    }

    resource ehub 'eventhubs@2022-01-01-preview' = {
        name: '${eventHubNamespaceName}-hub'
        properties: {
            messageRetentionInDays: 1
            partitionCount: 1
        }
        
        resource ehub_consumergroup 'consumergroups@2022-01-01-preview' = {
            name: eventHubConsumerGroupName
        }
    }
}
```